_posts/2022-10-13-septemeber-rubygems-updates.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Septemeber 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in September.
+
+## RubyGems News
+
+This month in RubyGems, we released RubyGems [v3.3.22](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3322--2022-09-07) and Bundler [v2.3.22](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2322-september-7-2022).
+
+The following are the main improvements shipped during this month (see the changelog for more improvements and fixes):
+
+- removed `auto-sudo` feature for installing gems - [#5888](https://github.com/rubygems/rubygems/pull/5888).
+- added `bundle console` man in Bundler's documentation - [#5901](https://github.com/rubygems/rubygems/pull/5901). 
+- updated the Bundler metadata source code URI for a more accurate source code uri in gemspec #5896
+https://github.com/rubygems/rubygems/pull/5896
+- removed warning for old TLS version connection - [#5928](https://github.com/rubygems/rubygems/pull/5928).
+- removed no longer needed `fiddle` hacks since RubyInstaller released patch versions to not load `fiddle` on boot - [#5902](https://github.com/rubygems/rubygems/pull/5902).
+
```

Comment:
I would add an extra paragraph here along the lines of

> In addition to the above, this month we've been working on migrating Bundler's internal resolver engine to use [Pub Grub](https://github.com/jhawthorn/pub_grub), which should result in much better error messages and resolution performance. We hope to release this work soon. Also, we have removed the feature of auto-sudo'ing when not having enough permissions to perform certain operations, because it was considered harmful and hardly useful. This removal will be released with Bundler 2.4.

Reply:
Thanks will include this.